### DISCUSSION TITLE
New site feedback - background animations

### DISCUSSION THREAD

**Discussion by solidgonk** [tteck/Proxmox#3093] [Ideas]
New site is neat - tidier than the previous one, and while it does take a few more seconds to load, then click through the 'Browse Scripts' button on page load, I can live with that.

However, the background animations are a bit distracting and are a little sea-sick inducing. I sometimes have the same issue with the parallax feature in iOS that moves the wallpaper slightly when you tilt the phone - there's a 'reduce motion' options to disable it.

Is it possible to add a button to disable all slide-ins, fade-ins and floaters on the new site?

**Comment by tteck** [comment]
> the background animations are a bit distracting and are a little sea-sick inducing.

Sorry about it making you a "little sea-sick". I'll look into an option.

> click through the 'Browse Scripts' button on page load

You can bookmark https://helper-scripts.com/scripts to avoid the landing page.
